### PR TITLE
update Telegram binding docs

### PR DESCRIPTION
**Pull request by mstormi**
add pointer to use bot to get Telegram chatID

### CODE REVIEW

File: bundles/org.openhab.binding.telegram/README.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ As described in the Telegram Bot API, this is the manual procedure needed in ord
 - Open a browser and invoke `https://api.telegram.org/bot<token>/getUpdates` (where `<token>` is the authentication token previously obtained)
 - Look at the JSON result to find the value of `id`. That is the chatId. Note that if using a Telegram group chat, the group chatIds are prefixed with a dash that must be included in the config file. (e.g. bot1.chatId: -22334455)
 
+If this does not work for you (the response may be empty), or you want to send *more* than one recipient (= chatID), the alternative is to contact (= open a chat with) a Telegram bot to respond with the chatID. There's a number of them such as @myidbot. Note they may work or not at any time so you eventually need to try another one.
```
Comment (Confectrician) [inline bundles/org.openhab.binding.telegram/README.md]: One line per sentence would be nice for keeping track of changes.

File: bundles/org.openhab.binding.telegram/README.md [hunk 2]
```diff
@@ -15,10 +15,14 @@ As described in the Telegram Bot API, this is the manual procedure needed in ord
 
 - Open a chat with your new Bot and send any message to it. The next step will not work unless you send a message to your bot first.
 
-3. Get the chatId
+3. Get the chatID
 
 - Open a browser and invoke `https://api.telegram.org/bot<token>/getUpdates` (where `<token>` is the authentication token previously obtained)
-- Look at the JSON result to find the value of `id`. That is the chatId. Note that if using a Telegram group chat, the group chatIds are prefixed with a dash that must be included in the config file. (e.g. bot1.chatId: -22334455)
+- Look at the JSON result to find the value of `id`. That is the chatID. Note that if using a Telegram group chat, the group chatIDs are prefixed with a dash that must be included in the config (e.g. `-22334455`).
```
Comment (J-N-K) [inline bundles/org.openhab.binding.telegram/README.md]: Please use a new line for each sentence

Comment (mstormi) [reply bundles/org.openhab.binding.telegram/README.md]: done

**Comment by mstormi** [comment]
it's a little annoying and quickly forgotten if you just find a quick to fix thingy and use the Github UI ... so here goes:

Signed-off-by: Markus Storm markus.storm@gmx.net

**Comment by J-N-K** [comment]
Sign-off in PR.

**Pull request merged**